private-templates-service/client/src/components/Common/DeleteModal/DeleteModal.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+// Libraries
+import { PrimaryButton } from 'office-ui-fabric-react';
+
+import { Template } from 'adaptive-templating-service-typescript-node';
+
+// Redux
+import { deleteTemplateVersion } from '../../../store/currentTemplate/actions';
+import { closeModal } from '../../../store/page/actions';
+
+// Components
+import AdaptiveCard from '../AdaptiveCard';
+import ModalHOC from '../../../utils/ModalHOC';
+
+// Strings
+import * as STRINGS from '../../../assets/strings';
+
+// Styles
+import {
+  CenterPanelWrapper,
+  BottomRow,
+} from '../../Common/UnpublishModal/styled';
+
+import {
+  Container,
+  ACWrapper,
+  TemplateName
+} from '../../AdaptiveCardPanel/styled'
+
+import {
+  BackDrop,
+  Modal,
+  Header,
+  Description,
+  DescriptionAccent,
+  ButtonGroup,
+  CancelButton,
+} from '../../Common/PublishModal/styled';
+
+
+interface Props {
+  template: Template;
+  templateVersion: string;
+  deleteTemplate: (templateVersion: string) => void;
+  closeModal: () => void;
+}
+
+const mapDispatchToProps = (dispatch: any) => {
+  return {
+    deleteTemplate: (templateVersion: string) => {
+      dispatch(deleteTemplateVersion(templateVersion));
+    },
+    closeModal: () => {
+      dispatch(closeModal());
+    }
+  }
+}
+
+class DeleteModal extends React.Component<Props> {
+  delete = () => {
+    this.props.deleteTemplate(this.props.templateVersion);
+    this.props.closeModal();
+  }
+
+  render() {
+    const { template } = this.props;
+
+    return (
+      <BackDrop>
+      <Modal>
+        <Header>Delete Template</Header>
+        <Description style={{ marginBottom: 0 }}>{STRINGS.DELETE_CONFIRMATION}<DescriptionAccent>{template.name} - {this.props.templateVersion}</DescriptionAccent>?</Description>
```

Comment:
Can styling be added to styled.tsx folder?

Reply:
Fixed, thanks!